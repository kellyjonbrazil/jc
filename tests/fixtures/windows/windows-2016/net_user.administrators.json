{
  "domain": "",
  "user_accounts": [
    {
      "user_name": "Administrator",
      "comment": "Built-in account for administering the computer/domain",
      "country_region_code": "000 (System Default)",
      "account_active": true,
      "account_expires": "Never",
      "password_last_set": "2021-12-16T11:07:14",
      "password_expires": "2022-01-27T11:07:14",
      "password_changeable": "2021-12-17T11:07:14",
      "password_required": true,
      "user_may_change_password": true,
      "workstations_allowed": "All",
      "last_logon": "2024-08-23T13:47:11",
      "logon_hours_allowed": "All",
      "local_group_memberships": [
        "Administrators"
      ],
      "global_group_memberships": []
    }
  ]
}
