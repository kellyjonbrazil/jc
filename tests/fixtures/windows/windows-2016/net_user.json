{
  "domain": "",
  "user_accounts": [
    {
      "user_name": "Administrator"
    },
    {
      "user_name": "DefaultAccount"
    },
    {
      "user_name": "Guest"
    },
    {
      "user_name": "user1"
    }
  ],
  "account_origin": "\\\\WIN-SERVER16"
}
