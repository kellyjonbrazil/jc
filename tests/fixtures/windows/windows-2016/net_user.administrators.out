User name                    Administrator
Full Name
Comment                      Built-in account for administering the computer/domain
User's comment
Country/region code          000 (System Default)
Account active               Yes
Account expires              Never

Password last set            12/16/2021 11:07:14 AM
Password expires             1/27/2022 11:07:14 AM
Password changeable          12/17/2021 11:07:14 AM
Password required            Yes
User may change password     Yes

Workstations allowed         All
Logon script
User profile
Home directory
Last logon                   8/23/2024 1:47:11 PM

Logon hours allowed          All

Local Group Memberships      *Administrators
Global Group memberships     *None
The command completed successfully.